### ISSUE TITLE
Reproducing the miou results of LCFCN

### ISSUE THREAD

**Issue by angrypiggy** [IssamLaradji/affinity_lcfcn#3]
Hi,

I tried to reproduce the miou result of LCFCN on the FishSeg test set. No matter how I tried, the best miou for the foreground I have got is around 62%, which is short from the paper results 68.4%. I am curious about the hyperparameter settings you used for this experiment, could you please kindly provide that? 

Currently I am using resnet38 as the backbone (pretrained on imagenet, I downloaded from this [repo](https://github.com/itijyou/ademxapp)), and I specified the backbone by assigning the "shared" argument True in the exp_dict, even though under the "model" fcn8_vgg16 is used. I use adam and beta (0.9, 0.999). I tried learning rate in {le-4, 1e-5, 1e-6}, and I also tried using a learning rate scheduler. For loss type, I used the lcfcn_loss. But I found there are lots of loss types provided too. I found lcfcn_const_loss can boost the performance too. I use RTX3090 to train these models. However, I still cannot reach the LCFCN performance of 68.4% for the foreground when training only using the point label of the FishSeg data.

Could you please provide more details about your experiment setting? Thank you so much!
